content/zh-tw/canary-deployment.md

```diff
@@ -0,0 +1,28 @@
+---
+title: 金絲雀部署
+status: Completed
+category: 概念
+tags: ["方法論", "應用程式", ""]
+---
+
+## 什麼是金絲雀部署 {#what-it-is}
```

Comment:
```suggestion
## 什麼是金絲雀部署 {#what-it-is}
```
I refer to the content of other zh-tw, it is recommended to be consistent

[1] content/zh-tw/cluster.md 
https://github.com/cncf/glossary/pull/2020/files#diff-db9b1dff7b96f1cf300424e6e661be236510ec7e79fae9b444516d4bc62892b9R8

[2] content/zh-tw/container-orchestration.md 
https://github.com/cncf/glossary/pull/2036/files#diff-262d0ef7001c7fbf22efaf69db9895573f4c02b02f672a93f4b3addc74f5bedbR7